data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/SchedulingParameterConfiguration.java

```diff
@@ -15,25 +15,25 @@
 
 public class SchedulingParameterConfiguration {
 
-    @JsonProperty("rate")
-    private Duration rate = Duration.ofHours(8);
+    @JsonProperty("interval")
+    private Duration interval = Duration.ofHours(8);
 
     @Min(1)
-    @JsonProperty("job_count")
-    private int jobCount = 1;
+    @JsonProperty("count")
```

Comment:
Is there some other noun modifier we can use instead of "job"? Count is vague and it is hard to understand what is happening here.

Reply:
It really represents the the number of times each index will be processed. Do you have a name suggestion? `index_processing_count` seems a little verbose

Reply:
It might be verbose, but I can understand it. Perhaps `index_read_count`? Or maybe `processing_count` to avoid the "per-index" idea.

Reply:
It is per index thought . I like `index_read_count`